packages/auto-label/package.json

```diff
@@ -18,7 +18,7 @@
     "probot-app"
   ],
   "scripts": {
-    "compile": "tsc -p . && cp ./src/colors.json ./build/src/colors.json && cp ./src/extensions.json ./build/src/extensions.json",
+    "compile": "tsc",
```

Comment:
Intentional update?

Reply:
Yes!  I updated the `tsconfig.json` to include JSON in it's compilation, allowing us to do the import style require like I did above.  This has the benefit of also copying the JSON files imported in source automatically into the `build/src` folder, negating the need for the `cp` commands.  Also `-p .` is the default behavior of `tsc`, so you know, saving some bits here and there 😛